tests/api/files/overdrive/audiobook.json

```diff
@@ -1 +1,177 @@
+{
```

Comment:
Maybe we should we leave this fixture stored without new-lines, since that is what gets served to us from the overdrive API?